v1.0/update.md

```diff
@@ -8,16 +8,399 @@ The `UPDATE` [statement](sql-statements.html) updates rows in a table.
+`RETURNING target_list` | Return values based on rows updated, where `target_list` can be specific column names from the table, `*` for all columns, or a computation on specific columns. <br><br>To return nothing in the response, not even the number of rows updated, use `RETURNING NOTHING`.
+
+## Examples
+
+### Update a Single Row
+
+~~~ sql
+> SELECT * FROM accounts;
+~~~
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   4000.0 | jim      |
+|  3 |   8700.0 | pam      |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+~~~ sql
+> UPDATE accounts SET balance = 5000.0 WHERE id = 2;
+
+> SELECT * FROM accounts;
+~~~
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   5000.0 | jim      |
+|  3 |   8700.0 | pam      |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+~~~ sql
+> UPDATE accounts SET (balance, customer) = (9000.0, 'kelly') WHERE id = 2;
+
+> SELECT * FROM accounts;
+~~~
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly   |
+|  3 |   8700.0 | pam      |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+~~~ sql
+> UPDATE accounts SET balance = 6300.0, customer = 'stanley' WHERE id = 3;
+
+> SELECT * FROM accounts;
+~~~
+
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly   |
+|  3 |   6300.0 | stanley  |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+### Update with default values
+
+~~~ sql
+> UPDATE accounts SET balance = DEFAULT where customer = 'stanley';
+
+> SELECT * FROM accounts;
+~~~
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly    |
+|  3 | NULL     | stanley  |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+### Update from a SELECT statement
+~~~ sql
+> UPDATE accounts SET (balance, customer) = (SELECT balance, customer FROM accounts WHERE id = 2) WHERE id = 4;
+
+> SELECT * FROM accounts;
+~~~
+
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly    |
+|  3 | NULL     | stanley  |
+|  4 |   9000.0 | kelly    |
++----+----------+----------+
+(4 rows)
+~~~
+
+### Update All Rows
+
+{{site.data.alerts.callout_danger}}If you don't use the WHERE clause to specify the rows to be updated, the values for all rows will be updated.{{site.data.alerts.end}}
```

Comment:
Need to apply code formatting to `WHERE`. Note that within a callout like this, you need to use html to format things:

`<code>WHERE</code>`